src/hooks/useCommand.js

```diff
@@ -1,35 +1,55 @@
+import MANUAL from "../data/manual";
 import useTerminal from "./useTerminal";
+import useMissions from "./useMissions";
+import useFlag from "./useFlag";
 
-// TODO: Implement commands
-export const commands = {
-  help: (argsList) => "Help command output.",
-  list: (argsList) => "List command output.",
-  select: (argsList) => "Select command output.",
-  status: (argsList) => "Status command output.",
-  flag: (argsList) => "List command output.",
-  clear: (argsList) => "Clear command output.",
-};
+const useCommand = () => {
+  const { stdIn, stdOut, clear } = useTerminal();
+  const { getMissions, selectMission, currentMission, getStatus } =
+    useMissions();
+  const { checkFlag } = useFlag(currentMission);
+
+  const handleHelp = () => stdOut(MANUAL);
+  const handleClear = () => clear();
+  const handleList = () => stdOut(getMissions());
+  const handleStatus = () => stdOut(getStatus());
+  const handleSelect = (args) => {
+    const id = parseInt(args[0]);
+    if (isNaN(id)) return stdOut(`err: invalid arg\n  select [MISSION ID]`);
+    stdOut(selectMission(id));
+  };
+  const handleFlag = (args) => {
```

Comment:
AI comment for this func. Of course is up to you ;)

_The handleFlag function should handle the case when the provided flag value is incorrect, for example:_ 
`if (!checkFlag(value)) return stdOut('Incorrect flag value!');`

Reply:
Obsłużone przez useFlag hook